Automattic/jetpack
# Cookies & Consent Widget: Default to core's privacy policy when present

Also includes some updates to the widget form for clarity.

Begins work on #9512

#### Changes proposed in this Pull Request:

* When a user site has a Privacy Policy page set (introduced in 4.9.6), default to using that privacy policy as a custom policy URL.
* Update some of the widget form labels for clarity

#### Testing instructions:

Test using the WP 4.9.6 beta with the following scenarios:
* With no privacy policy set in WP core, add a new widget and make sure it defaults to using the "default" policy url.
* Set a privacy policy in WP Core. Then add a new widget and make sure the policy URL defaults to a custom URL which is pre-populated with the privacy policy page URL set in WP Core.
